renovate.json

```diff
@@ -20,6 +20,10 @@
     "performance"
   ],
   "packageRules": [
+    {
+      "updateTypes": ["minor", "patch"],
+      "automerge": true
+    },
```

Comment:
I can't very if this works without merging, but there is a simpler way. See https://github.com/aerogear/graphback/commit/fefa6f83c65acf2651459b2a028d28f3bbc9ad67#diff-7b5c8955fc544a11b4b74eddb4115f9cc51c9cf162dbffa60d37eeed82a55a57

Reply:
Okay, I will do the same.